libs/utils/android/workloads/uibench.py

```diff
@@ -84,6 +84,11 @@ def run(self, out_dir, test_name, duration_s, collect=''):
         # Keep track of mandatory parameters
         self.out_dir = out_dir
         self.collect = collect
+        # For systrace collection, the duration is expected to be part of `collect` parameter
+        # This is because systrace capturing currently doesnt support CTRL+C stopping
+        # so tracingStop() call just waits instead.
+        if collect == 'systrace':
+            self.collect = 'systrace_' + str(duration_s)
```

Comment:
That would make sense if only systrace was supported.
What happens when the user chooses ftrace instead? That does support stopping after n seconds.
By removing duration_s, I assume you then expect them to format their 'collect' parameter to be <type>_<duration> for both ftrace and systrace (and any future supported methods)?